src/WebApi/WebApi/UseCases/v1/Question/UpdateQuestion/UpdateQuestionRequest.cs

```diff
@@ -7,7 +8,7 @@ public record UpdateQuestionRequest
     [MaxLength(100)]
     public string Title { get; set; }
 
-    [Range(1, 5)]
+    [Range(0, 100, ErrorMessage = "The Difficulty field is out of range ( 0 - 100 )")]
```

Comment:
@Ali-Kabbadj you're also failing a test here, since this **used to be** 1 - 5 and you made it 0 - 100

```suggestion
    [Range(1, 5)]
```

So with this one, I'd just put it back

Reply:
@TomasNiemczyk98 what does that really mean in that case? What should really be the range here?

Reply:
The range should be the accepted values for our difficulty field, it was my understanding that we are switching from a range of 1-5 to a range of 0-100